contrib/functions/go/generate-kpt-pkg-docs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine3.13
+FROM golang:1.17-alpine3.13
```

Comment:
What would you think about defining common Dockerfile(s) for all of the functions? It seems like if our intent is to keep the dependencies (e.g. language version, base image, etc) consistent, then it might make this easier to manage/maintain if we reused common Dockerfile(s).

Another option might be to pass in the base image as an argument to the Dockerfile(s), so that we can define the value(s) in one place

I'd say this suggestion is out of scope for this issue, but I was just curious of your thoughts